packages/web/src/components/buy-audio-modal/components/InProgressPage.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  buyAudioSelectors,
+  BuyAudioStage,
+  formatNumberString
+} from '@audius/common'
+import { IconCaretDown } from '@audius/stems'
+import { useSelector } from 'react-redux'
+
+import LoadingSpinner from 'components/loading-spinner/LoadingSpinner'
+import { ToggleCollapseButton } from 'components/toggle-collapse-button'
+
+import { IconAUDIO, IconSOL, IconUSD } from './Icons'
+import styles from './InProgressPage.module.css'
+
+const { getAudioPurchaseInfo, getBuyAudioFlowStage } = buyAudioSelectors
+
+const messages = {
+  pleaseHold: 'Please hold on. This may take a few moments.',
+  completeWithCoinbase: 'Complete this step with Coinbase',
+  step1: 'Step 1',
+  step2: 'Step 2',
+  purchasingSol: 'Purchasing SOL',
+  convertingSol: 'Converting SOL to $AUDIO',
+  moreInfo: 'More Info',
+  lessInfo: 'Less Info',
+  usd: 'USD',
+  sol: 'SOL',
+  audio: '$AUDIO'
+}
+
+export const InProgressPage = () => {
```

Comment:
are these pages? i think it's fine but wonder if there is a good alternative?

Reply:
how about `InProgressModalPage`?

Reply:
right... well it's technically modal-content right? so maybe that?

Reply:
but also down for `InProgressContent`?

Reply:
sure, can do that!

Reply:
actually how about InProgressPageContent? they really are pages of the modal in my opinion

Reply:
I'm going to leave this as Page for now, can revisit!!!